config/routes.rb

```diff
@@ -1,6 +1,7 @@
 Rails.application.routes.draw do
-  # Define your application routes per the DSL in https://guides.rubyonrails.org/routing.html
+  root 'users#index'
 
-  # Defines the root path route ("/")
-  # root "articles#index"
+  resources :users, only: [:show] do
+    resources :posts, only: [:index, :show]
+  end
```

Comment:
- [ ] As required for the project, kindly ensure that you have all URLs required for the this milestone. Right now. I notice that you miss route for the `users#index` page. Make sure you add it, i.e.
```ruby
  root 'users#index'

  resources :users, only: [:index, :show] do
    resources :posts, only: [:index, :show]
  end
```